### PR TITLE
Edited Hub Entry

### PR DESCRIPTION
**Pull request by 13spacemen**
## About The Pull Request
Adds round time and alert level to the hub entry
Removed redundant "secret" and player number, after the server name

## Why It's Good For The Game
Helps players make informed decisions on whether to join the server or not, also it's a dynamic element of the hub entry so makes the server feel alive on the hub

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
add: The BYOND hub entry shows time and alert now
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — DISMISSED** [review]
@Crossedfall

**Comment by Crossedfall** [comment]
Eeh. I'm torn. I'm kind of against this. Sometimes there is such a thing as too much information. It's kind of nice to get people to hop on to check things out, even if it results in only a small percentage of people actually choosing to stick it out.

I think I'd rather have people test the waters over avoiding the round entirely just because of a map or whatever.

**Comment by 13spacemen** [comment]
@Crossedfall I can limit it to the round time only if you want. To me that's the most important thing, SS13 is pretty time-consuming so being able to see how far in the current round is at a glance is useful

File: code/game/world.dm [hunk 1]
```diff
@@ -357,6 +344,10 @@ GLOBAL_VAR(restart_counter)
 	if (features)
 		s += ": [jointext(features, ", ")]"
 
+	s += "<br>Time: <b>[gameTimestamp("hh:mm")]</b>"
+	s += "<br>Alert: <b>[capitalize(get_security_level())]</b>"
+	s += "<br>Roleplay: <b>Medium</b>
```
Comment (PowerfulBacon) [inline code/game/world.dm]: There are multiple reasons why we can't have this:
 - Hardcoding values that downstreams don't follow
 - We don't advertise as a medium roleplay server, just a roleplay server.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Use the normal PR template.

File: code/game/world.dm (hunk 1 above)
Comment (13spacemen) [reply code/game/world.dm]: alright will remove roleplay line

File: code/game/world.dm [hunk 2]
```diff
@@ -335,28 +332,21 @@ GLOBAL_VAR(restart_counter)
...
 
 	if (!host && hostedby)
 		features += "hosted by <b>[hostedby]</b>"
 
-	if (features)
+	if(length(features))
 		s += ": [jointext(features, ", ")]"
 
+	s += "<br>Time: <b>[gameTimestamp("hh:mm")]</b>"
+	s += "<br>Alert: <b>[capitalize(get_security_level())]</b>"
```
Comment (PowerfulBacon) [inline code/game/world.dm:353]: I think players & popcap should be included in here (Useful to know if the server is full and how much space is left)

Comment (13spacemen) [reply code/game/world.dm:353]: @PowerfulBacon done

Comment (Crossedfall) [reply code/game/world.dm:353]: Popcap is already included. It's a basic hub feature. 
image

File: code/game/world.dm [hunk 3]
```diff
@@ -344,19 +341,18 @@ GLOBAL_VAR(restart_counter)
...
+	s += "<br>Players: <b>[players][popcaptext]</b>"
```
Comment (Crossedfall) [inline code/game/world.dm:354]:
```suggestion
```

Unneeded.

File: code/game/world.dm (hunk 2 above)
Comment (Crossedfall) [reply code/game/world.dm:353]: Disregard, misunderstood the request

File: code/game/world.dm (hunk 3 above)
Comment (Crossedfall) [reply code/game/world.dm:354]: Disregard

**Review by PowerfulBacon — APPROVED** [review]

**Review by itsmeow — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**